### PR TITLE
[Admin][Order] Edit order billing address

### PR DESCRIPTION
**Pull request by tuka217**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | no|
| Related tickets | - |
| License         | MIT |

### CODE REVIEW

File: features/order/managing_orders/modifying_billing_address/modifying_billing_address.feature [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    @ui
+    Scenario: Modifying a customer's billing address when a channel has been disabled
+        Given the channel "Web" has been disabled
+        When I view the summary of the order "#00000001"
+        And I want to modify a customer's billing address of this order
+        And I specify their new billing address as "Los Angeles", "Seaside Fwy", "90802", "United States" for "Lucifer Morningstar"
+        And I save my changes
+        Then I should be notified that it has been successfully edited
+        And this order bill should be shipped to "Lucifer Morningstar", "Seaside Fwy", "90802", "Los Angeles", "United States"
+        And the order's total should still be "$400.00"
```
Comment (lchrusciel) [inline features/order/managing_orders/modifying_billing_address/modifying_billing_address.feature]: No need for this assertion. Here and below

File: src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php [hunk 2]
```diff
@@ -185,10 +185,15 @@ public function itShouldBeShippedTo(OrderInterface $order = null, $customerName,
     }
 
     /**
+     * @Then this order bill should be shipped to :customerName, :street, :postcode, :city, :countryName
```
Comment (lchrusciel) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php]: Is it even working? `/^(this order) bill should be shipped to "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)"$/`

File: features/order/managing_orders/modifying_billing_address/modifying_billing_address.feature (hunk 1 above)
Comment (NeverResponse) [inline features/order/managing_orders/modifying_billing_address/modifying_billing_address.feature]: No, it's just irrelevant. The point is to check the address, so the only thing we need to do is check the address. 🍾

Comment (tuka217) [inline features/order/managing_orders/modifying_billing_address/modifying_billing_address.feature]: This scenario want to check if after changing address the price was the same, although the settings on shop was changed. So the price should be checked.

**Comment by pjedrzejewski** [comment]
Thank you Ania!

**Pull request merged**